src/utils/object/omit.test.ts

```diff
@@ -0,0 +1,14 @@
+import { omit } from "utils/object";
+
+describe("omit", () => {
+  it("returns a new object without the supplied key", () => {
+    expect(omit<{ b?: number }, ["b"]>({}, ["b"])).toMatchObject({});
+    expect(omit({ a: 1 }, ["a"])).toMatchObject({});
+  });
+  it("returns a new object reference even when the input and outupt object have deep equality", () => {
```

Comment:
```suggestion
  it("returns a new object reference even when the input and output object have deep equality", () => {
```

Reply:
This test is a bit confusing is it trying to test that the input and output are the same? 
It doesn't even actually test out the functionality since its not actually removing any fields from inputObj.